src/askem_beaker/contexts/model_configuration/context.py

```diff
@@ -119,6 +119,15 @@ async def load_config(self):
         print(f"Running command:\n-------\n{command}\n---------")
         await self.execute(command)        
 
+    async def post_execute(self, message):
+        try:
+            content = (await self.evaluate(self.get_code("get_config")))["return"]
+            self.beaker_kernel.send_response(
+                "iopub", "model_configuration_preview", content, parent_header=message.parent_header
+            )
+        except Exception as e:
+            raise
```

Comment:
```suggestion
        content = (await self.evaluate(self.get_code("get_config")))["return"]
        self.beaker_kernel.send_response(
            "iopub", "model_configuration_preview", content, parent_header=message.parent_header
        )
```
Feel free to ignore but maybe this could simplify a little bit.

Reply:
thanks!